images/win/scripts/Installers/Install-Chrome.ps1

```diff
@@ -2,25 +2,116 @@
 ##  File:  Install-Chrome.ps1
 ##  Desc:  Install Google Chrome
 ################################################################################
+function Stop-SvcWithErrHandling
+{
+    param (
+        [Parameter(Mandatory, ValueFromPipeLine = $true)] [string] $ServiceName
+    )
 
-Import-Module -Name ImageHelpers -Force
+    Begin { 
+        Write-Debug "Function [Stop-SvcWithErrHnadlig] is started"; 
+    }
+    Process {
+        $Service = Get-Service $ServiceName -ErrorAction SilentlyContinue
+        if(-not $Service) {
+            Write-Warning "[!] Service [$ServiceName] is not found";
+        }
+        else {
+            Write-Debug "Try to stop service [$ServiceName]";
+            try {
+                Stop-Service -Name $ServiceName -Force;
+                $Service.WaitForStatus("Stopped", "00:01:00");
+                Write-Debug "Service [$ServiceName] has been stoppet successfuly";
+            }
+            catch {
+                Write-Error "[!] Failed to stop service [$ServiceName] with error:"
+                $_ | Out-String | Write-Error;
+            }
+        }
+    }
+    End {
+        Write-Debug "Function [Stop-SvcWithErrHnadlig] is stopped";
```

Comment:
typo in `handling`

Reply:
'Begin' and 'End' sections have been removed from functions